tests/installation/release_notes_from_rpm.pm

```diff
@@ -1,36 +1,31 @@
 # SUSE's openQA tests
 #
-# Copyright © 2017-2018 SUSE LLC
+# Copyright © 2021 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
 # Summary: Check origin of displayed releasenotes during installation
```

Comment:
s/releasenotes/release notes/
It doesn't check the origin, check that the origin is from rpm. Please also update the other file.